src/Main.elm

```diff
@@ -946,8 +962,24 @@ viewFormBuilder maybeAnimate { dropdownState, formFields, shortTextTypeList } =
         extraOptions =
             shortTextTypeList
                 |> List.map
-                    (\( k, _ ) ->
-                        ( AddFormField (ShortText k Nothing)
+                    (\( k, v ) ->
+                        let
+                            maybeMaxLength =
+                                Dict.get "maxlength" v
+
+                            maybeMultiple =
+                                Dict.get "multiple" v
+
+                            toTuple : String -> Maybe String -> ( String, String )
+                            toTuple key val =
+                                ( key, Maybe.withDefault "" val )
+                        in
+                        ( AddFormField
+                            (ShortText k
+                                [ toTuple "maxlength" maybeMaxLength
+                                , toTuple "multiple" maybeMultiple
+                                ]
```

Comment:
I'm not sure why we're specifically calling out `maxlength` and `multiple` instead of converting entire `Dict String String` to `List (String, String)` https://package.elm-lang.org/packages/elm/core/latest/Dict#toList

Reply:
sorry, the PR was initially to add only `multiple`, so my train of thoughts keep going there 😅